### ISSUE TITLE
External HDD & bottom controls

### ISSUE THREAD

**Issue by nmahir** [thekirankumar/carstream-android-auto#159]
Hi,

Just wondering is it possible to have an external hdd plugged in and be able to play videos via carstream?
Also is it possible to remove the bottom android auto controls when a video is playing.

Thanks for your help.

**Comment by air-hun** [comment]
And where do you plug the external HDD in?

**Comment by nmahir** [comment]
Into the usb port in the car.

**Comment by air-hun** [comment]
How many USB ports do you have? I have only one. It needed for connecting the phone to car's HU.

**Comment by nmahir** [comment]
I have 2 usb ports

**Comment by air-hun** [comment]
Which car? Both works together?

**Comment by nmahir** [comment]
Its a 2017 ford focus sport, im not sure how you mean works together

**Comment by air-hun** [comment]
One for linking the phone with the car via Android Auto, and one for mass storage drives. And car's HU can handle both devices together in same time?

**Comment by nmahir** [comment]
Yes just tried. I can use android auto from phone and exit and see the hdd.

**Comment by air-hun** [comment]
Ok. If you have to exit from Android Auto to use the HDD that means HU can't handle both USB ports together. :(

**Comment by air-hun** [comment]
I think one USB port just used for charging if the other one is used for linking.

**Comment by nmahir** [comment]
How can I try using both at the same time?

**Comment by shmykelsa** [comment]
Well, the HDD sould pass through the phone, so no chances of that happening while using Android Auto.

Bottom bar, as stated on the website, cannot be hidden
